package.json

```diff
@@ -3,6 +3,9 @@
   "version": "1.1.1",
   "description": "rnpm plugin that links native dependencies to your project",
   "main": "index.js",
+  "scripts": {
+    "test": "eslint ./ && mocha ./test --recursive"
```

Comment:
Well, I have something to add here. I insist to use local dependencies over global ones. I think I don't need to explain "why?". So I propose to change it to something like this: 
```bash
node_modules/eslint/bin/eslint . && ./node_modules/mocha/bin/mocha
```
(./test --recursive is a default settings afaik)